src/components/AddBook/index.js

```diff
@@ -118,11 +131,14 @@ function AddBook() {
           placeholder="None selected"
           onChange={changeGenre}
         >
-          <option value="">Select an option</option>
-          <option value="1">Thrlller</option>
-          <option value="2">Romance</option>
-          <option value="3">Historical</option>
-          <option value="4">Non-fiction</option>
+          <option className="dropdown" value={0}>
+            All
+          </option>
+          {bookGenres.map((genre) => (
+            <option className="dropdown" value={genre.id} key={genre.id}>
+              {genre.name}
+            </option>
+          ))}
         </select>
```

Comment:
I would suggest moving this into it's own component, along with the actual fetching of the genres. Trying to keep each component as small as possible.